### DISCUSSION TITLE
EPUB reader continuous scrolling

### DISCUSSION THREAD

**Discussion by chong206** [gotson/komga#1690] [Ideas]
When using scrolling mode for EPUB format e-books, scrolling will result in card sticking.
And it will not automatically jump to the next chapter and requires clicking. Can you add a feature that allows continuous scrolling like the comic Webtoon mode? Thank you.

**Comment by gotson** [comment]
> scrolling will result in card sticking.

what does that mean?

> And it will not automatically jump to the next chapter and requires clicking. Can you add a feature that allows continuous scrolling like the comic Webtoon mode? Thank you.

this is not possible with the current epub reader

**Reply by chong206** [reply]
I keep scrolling, but it's not smooth and there are occasional pauses. I recorded a video

https://github.com/user-attachments/assets/b3b3b94f-0d20-4b55-9ec6-6348f9f3a62a



If possible, please request to add this feature.thanks

Question 2
I turned on the reverse proxy of Synology and when I opened EPUB, it showed blank.
Using HTTP to access EPUB, reading can be accessed normally.

![截图_20240912194039](https://github.com/user-attachments/assets/686e97ae-1dce-473a-b740-0437d59c77bc)
![截图_20240912194247](https://github.com/user-attachments/assets/dd4023cc-be82-4fa6-80c2-ea4286a9a4e3)
![1](https://github.com/user-attachments/assets/001a044f-af61-4c71-9010-fbad3fe5be12)

**Reply by gotson** [reply]
> Question 2 I turned on the reverse proxy of Synology and when I opened EPUB, it showed blank. Using HTTP to access EPUB, reading can be accessed normally.

your reverse proxy does not correctly set headers like `X-Forwarded-Proto`

**Comment by chong206** [comment]
@gotson I am certain that the reverse proxy has correctly set the header.
It was found that the nested or called URLs in the project are all HTTP URLs. If using HTTPS to access a website, all nested or called URLs inside cannot be HTTP. Therefore, using HTTP access is normal, but using HTTPS access is not normal.
<img width="590" alt="4b4061606288fb9e1b63fdce3f536c9" src="https://github.com/user-attachments/assets/71902914-2eef-4e73-b9af-c2bc7752fa04">
![截图_20240918215914](https://github.com/user-attachments/assets/97e49d62-5175-4a26-82de-e0a0c2e0fe15)
